install/upgrade-postgres.sh

```diff
@@ -24,14 +24,14 @@ if [[ -n "$(docker volume ls -q --filter name=sentry-postgres)" && "$(docker run
 
   # Wait for postgres
   RETRIES=5
-  until docker exec sentry-self-hosted-postgres-1 psql -U postgres -c "select 1" >/dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
+  until docker compose exec postgres psql -U postgres -c "select 1" >/dev/null 2>&1 || [ $RETRIES -eq 0 ]; do
     echo "Waiting for postgres server, $((RETRIES--)) remaining attempts..."
     sleep 1
   done
 
   # VOLUME_NAME is the same as container name
   # Reindex all databases and their system catalogs which are not templates
-  DBS=$(docker exec sentry-self-hosted-postgres-1 psql -qAt -U postgres -c "SELECT datname FROM pg_database WHERE datistemplate = false;")
+  DBS=$(docker compose exec postgres psql -qAt -U postgres -c "SELECT datname FROM pg_database WHERE datistemplate = false;")
   for db in ${DBS}; do
     echo "Re-indexing database: ${db}"
     docker exec sentry-self-hosted-postgres-1 psql -qAt -U postgres -d ${db} -c "reindex system ${db}"
```

Comment:
How about this one and the one below too 😄

Reply:
done 👍